GitUI/GitUICommands.cs

```diff
@@ -1431,7 +1431,7 @@ public bool RunCommand(IReadOnlyList<string> args)
                 return false;
             }
 
-            if ((command == BlameHistoryCommand || command == FileHistoryCommand) && args.Count <= 2)
+            if ((command is BlameHistoryCommand or FileHistoryCommand) && args.Count <= 2)
```

Comment:
```suggestion
            if (args.Count <= 2 && (command is BlameHistoryCommand or FileHistoryCommand))
```

Reply:
I prefer the form already in code. The condition which restricts the argument count is the command, so it should be checked first unless fast-path performance dictates otherwise (which is not a problem in this case).